controller/comments.controller.js

```diff
@@ -0,0 +1,20 @@
+const { fetchCommentsByArticleId } = require('../models/comments.model');
+
+exports.getCommentsByArticleId = (req, res, next) => {
+const { article_id } = req.params
+
+  if (isNaN(article_id)) {
+    return res.status(400).send({ msg: 'invalid article id' })
+  }
```

Comment:
You don't need to manually check for the article id not being a number - if you try to do a query with a string instead of a number psql will throw an error - you can then use that error instead of a manual one